scss/components/_list-styles.scss

```diff
@@ -42,6 +43,11 @@
   }
 }
 
+// Alternate list styles
+ul.row {
```

Comment:
It's on the beta home page around the bulleted list. I couldn't find class="row" in other files to edit, but maybe I missed it!